README.md

```diff
@@ -160,7 +168,8 @@ In Windows, the script is generally installed in
     C:\>\Python34\python \Python34\Scripts\mypy PROGRAM
 
 If you are on Windows using Python 3.3 or 3.4, and would like to use XML reports
-download LXML from [PyPi](https://pypi.python.org/pypi/lxml).
+download LXML from [PyPi](https://pypi.python.org/pypi/lxml). 
+XML based reports do not work on Python 3.3 and 3.4 on Windows.
```

Comment:
They do work, but only if you have LXML installed. The last sentence can be deleted.